2996_reflection/reflection.md

```diff
@@ -4972,6 +5171,22 @@ Defines `class_type` with properties as follows:
 :::
 :::
 
+### [meta.reflection.static_string] Static string generation  {-}
+
+::: std
+::: addu
+```cpp
+consteval const char *define_static_string(std::string_view str);
+consteval const char8_t *u8define_static_string(std::u8string_view str);
+```
+
+[#]{.pnum} *Returns*: A pointer to the first element of an array of `char` or `char8_t` objects, respectively, whose length is `str.size() + 1`. The values of the first `str.size()` array subobjects are `str[k]`, and the value of the last array subobject is `0`. The array has static storage duration, and is usable in constant expressions.
```

Comment:
I think more like:

* Let `A` be a variable with static storage duration of type `char[str.size()]` or `char8_t[str.size()]` respectively, such that `A[i] == str[i]` for all 0 &le; `i` &lt; `str.size()` and `A[str.size()] == '\0'`. 

* Returns: `A`

Reply:
Thanks, done. Also specified that `A` is constexpr, which I think is needed to ensure that it's "usable in constant expressions".